package.json

```diff
@@ -39,6 +39,7 @@
   },
   "devDependencies": {
     "eslint": "^7.20.0",
+    "eslint-plugin-cypress": "^2.11.2",
```

Comment:
Also add it to `peerDependencies`. Technically it doesn't need to be in `devDependencies`, but we might as well keep the two in sync.